documentation/src/docs/pravega-concepts.md

```diff
@@ -11,362 +11,250 @@ You may obtain a copy of the License at
-no longer accepts writes.  At this point in time, Events with Routing Key
-300 and above are written to Segment 3 and those between 200 and 299 would be
-written into Segment 2.  Segment 0 still keeps accepting the same range of
-Events as before t1.  
-
-We also see another Scale-up event at time t2, as Segment 0’s range of Routing
-Key is split into Segment 5 & Segment 4.  Also at this time, Segment 0 is sealed
-off so that it accepts no further writes.
-
-Segments covering a contiguous range of the key space can also be merged.  At
-time t3, Segment 2’s range and Segment 5’s range are merged into Segment 6 to
-accommodate a decrease in load on the Stream.
-
-When a Stream is created, it is configured with an Scaling Policy that
-determines how a Stream reacts to changes in its load.  Currently there are
-three kinds of Scaling Policy:
-
-1.  Fixed.  The number of Stream Segments does not vary with load
-
-2.  Size-based.  As the number of bytes of data per second written to the Stream
-    increases past a certain target rate, the number of Stream Segments is
-    increased.  If it falls below a certain level, decrease the number of Stream
-    Segments.
-
-3.  Event-based.  Similar to Size-based Scaling Policy, except it uses the
-    number of Events instead of the number of bytes.
+- A Stream starts at time **t0** with a configurable number of Segments.  If the
+rate of data written to the Stream is constant, there will be no change in the number of Segments. 
+
+- At time **t1**, the system noted an increase in the ingestion rate and splits **Segment 1** into two parts. This process is referred as **Scale-Up** Event.
+
+- Before **t1**, Events with a Routing Key that hashes to the upper part of the key
+space (i.e., values ranging from **200-399**) would be placed in **Segment 1** and those that hash into the
+lower part of the key space (i.e., values ranging from **0-199**) would be placed in **Segment 0**.
+
+- After **t1**, **Segment 1** is split into **Segment 2** and **Segment 3**. The **Segment 1** is sealed and stops accepting writes.  At this point in time, Events with Routing Key **300** and _above_ are written to **Segment 3** and those between **200** and **299** would be written into **Segment 2**.
+
+- **Segment 0** continues accepting the same range of Events as before **t1**.  
+
+- Another scale-up Event occurs at time **t2**, as **Segment 0**’s range of Routing
+Key is split into **Segment 5** and **Segment 4**. Also at this time, **Segment 0** is sealed
+and allows no further writes.
+
+- Segments covering a contiguous range of the key space can also be merged. At
+time **t3**, **Segment 2**’s range and **Segment 5**’s range are merged to **Segment 6** to
+accommodate a decrease in the load on the Stream.
+
+When a Stream is created, it is configured with a **Scaling Policy** that
+determines, how a Stream handles the varying changes in its load. Pravega has three kinds of Scaling Policy:
```

Comment:
Please, delete the `,` between `determines, how`.

Reply:
done